src/speechly/client.ts

```diff
@@ -61,6 +61,8 @@ export class Client {
   private readonly activeContexts = new Map<string, Map<number, SegmentState>>()
   private readonly reconnectAttemptCount = 5
   private readonly reconnectMinDelay = 1000
+  private readonly contextStopDelay = 250
```

Comment:
How about adding this to `ClientOptions` as an optional property? I think it will make it easier to experiment with different delay values in case 250 millisecond is not good enough.

Reply:
Let's separate this task.